data/team.yml

```diff
@@ -40,7 +29,7 @@ members:
     position: Head of People Operations
     bio: >
       Loves people, learning, and to travel. Thrives in helping the team shoot
-      for the moon. Believes everyone can achieve their goals and dreams. 
+      for the moon. Believes everyone can achieve their goals and dreams.
```

Comment:
As an outside contributor, you are not entitled to change other member's bios

Shame on you

![https://www.getdigital.eu/web/getdigital/gfx/products/__generated__resized/380x380/Aufkleber_Trollface.jpg](https://www.getdigital.eu/web/getdigital/gfx/products/__generated__resized/380x380/Aufkleber_Trollface.jpg)